packages/react-devtools-core/webpack.backend.js

```diff
@@ -58,6 +58,9 @@ module.exports = {
       'process.env.GITHUB_URL': `"${GITHUB_URL}"`,
     }),
   ],
+  optimization: {
+    minimize: false,
+  },
```

Comment:
Technically unrelated but I got tired of the unreadable bundle so I changed this to match the inline package's settings.

This code (DevTools backend) is either (a) only lazy-loaded in the browser on demand or (b) only embedded in RN DEV builds so it shouldn't matter.

Reply:
Does it keep optimizations on? Like removing dead code. It would be nice to keep all optimizations on while disabling the renaming / whitespace cutting passes.

Reply:
Tested quickly by adding to the source:
```js
if (__DEV__) {
  console.log('__DEV__');
} else {
  console.log('!__DEV__');
}
```
Compiled bundle was:
```js
if (false) {} else {
  console.log('!__DEV__');
}
```